README.md

```diff
@@ -4,7 +4,7 @@
 
 ## What is Rafiki?
 
-Rafiki is open source software that allows an [Account Servicing Entity](https://rafiki.dev/concepts/account-servicing-entity/) to enable [Interledger](https://rafiki.dev/concepts/interledger-protocol/overview/) functionality on its users' accounts.
+Rafiki is open source software that provides a more efficient method for an [Account Servicing Entity](https://rafiki.dev/concepts/account-servicing-entity/) to enable Interledger functionality on its users' accounts.
```

Comment:
```suggestion
Rafiki is open source software that provides an efficient solution for an [Account Servicing Entity](https://rafiki.dev/concepts/account-servicing-entity/) to enable Interledger functionality on its users' accounts.
```
nit-picky, but "a more" implies some sort of comparison between an alternative which we don't talk about

Reply:
Should we also update the landing page?

Reply:
good point. both of them.